aws/cluster.tf

```diff
@@ -6,6 +6,12 @@ locals {
   uuid = random_uuid.cluster.result
   timestamp = time_static.timestamp.rfc3339
   deployment_id = "redpanda-${local.uuid}-${local.timestamp}"
+
```

Comment:
Since we use the same tags on every resources, I factored them out into a local.